### DISCUSSION TITLE
Generating a model instance from json() output

### DISCUSSION THREAD

**Discussion by zopyx** [pydantic/pydantic#3650] [Question]
Pydantic support the serialization to JSON using the  `json()` method ("export").

Is there a pendant as reverse operation for generating a model instanct from an exported JSON string?

The related `dataclasses_json` Python module provide such a round-tripping.

**Comment by uriyyo** [comment]
You can use `parse_raw` classmethod:
```py
from pydantic import BaseModel


class Model(BaseModel):
    a: str
    b: str


m = Model(a="1", b="2")
print(m)

s = m.json()
print(Model.parse_raw(s))
```
```py
a='1' b='2'
a='1' b='2'
```
